src/main/java/frc/robot/subsystems/arm/ArmCal.java

```diff
@@ -48,5 +48,6 @@ public class ArmCal {
   /** Various timeouts for auto lift movements */
   public static final double SCORE_TO_START_FAST_SEC = PLACEHOLDER_DOUBLE,
       SCORE_TO_START_SEC = PLACEHOLDER_DOUBLE,
-      START_TO_PRESCORE_HIGH_SEC = PLACEHOLDER_DOUBLE;
+      START_TO_PRESCORE_MID_HIGH_SEC = PLACEHOLDER_DOUBLE,
+      START_TO_PRESCORE_LOW_SEC = PLACEHOLDER_DOUBLE;
```

Comment:
Can you make the arbitrarily large like a couple seconds or now? And we'll make smaller if we need to